addons/attributes/initAttributes.sqf

```diff
@@ -185,6 +185,21 @@
     {alive _entity && {_entity isKindOf "LandVehicle" || {_entity isKindOf "Air"} || {_entity isKindOf "Ship"}}}
 ] call FUNC(addAttribute);
 
+[
+    "Object",
+    [LSTRING(BuildingMarker), LSTRING(BuildingMarker_Tooltip)],
+    QGVAR(icons),
```

Comment:
I think a toolbox or combo box would work better for this.

Reply:
https://github.com/zen-mod/ZEN/pull/704/commits/0ac3c6a9e07a18dd15e2c022fe5308b5576f73b9#diff-ab311763189315d2a6fea02999d8c7652230bab53f6b59959e3ac16ffdbfd10aR191